.vscode/settings.json

```diff
@@ -1,3 +1,3 @@
 {
```

Comment:
- [ ] As you guys know that this is a pair-programming project but i cannot see any commit from @TayyabKM. My humble request is from @TayyabKM to make some contributions in this pull request to follow the requirements and submit this pull request again from the dashboard for a review 👍